settings-schema.json

```diff
@@ -97,18 +103,18 @@
                 }
             }
         }
-        ,"grouping_initial":
-            {"title": "Initial ID grouping"
-            ,"description": "initial ID grouping variable"
-            ,"default": null
-            ,"type": "string"
-            }
-        ,"grouping_direction":
-            {"title": "Grouping annotation placement"
-            ,"description": "grouping annotation placement, 'horizontal' or 'vertical'"
-            ,"default": "horizontal"
-            ,"type": "string"
-            }
+    ,"grouping_initial":
+        {"title": "Initial ID grouping"
+        ,"description": "initial ID grouping variable"
+        ,"default": null
+        ,"type": "string"
+        }
+    ,"grouping_direction":
```

Comment:
Add enum?

Reply:
Yeah, I'll make sure to update the settings-schema after everything is merged.  Good idea to do this before each new release.  Addressed in #116.